From: Jason Rhinelander <jason@imaginary.ca>
Date: Wed, 8 Jan 2020 20:44:37 -0400
Subject: Enable cpu detection on amd64

The optional runtime detection shouldn't rely on being compiled in avx2
mode.
---
 crypto/libntrup/src/ntru.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/libntrup/src/ntru.cpp b/crypto/libntrup/src/ntru.cpp
index c86451b..fd8f620 100644
--- a/crypto/libntrup/src/ntru.cpp
+++ b/crypto/libntrup/src/ntru.cpp
@@ -1,6 +1,6 @@
 #include <libntrup/ntru.h>
 
-#if __AVX2__
+#ifdef __x86_64__
 #include <cpuid.h>
 #include <array>
 
